.gitlab-ci.yml

```diff
@@ -202,6 +205,7 @@ deploy:sls:production:
     - build:native
     - test:unit
     - test:browser
+    - test:build-output
     - sync:squidex:production
```

Comment:
Ideally we would also require `verify:development` here so that master runs a sequential
`deploy dev - verify dev - deploy prod - verify prod` and prod deployment is prevented if dev verification fails. But requires a bit of stage restructuring

Reply:
Also totally not related to the change I'm making

Reply:
yeah that the plan, see https://trello.com/c/FMU3iWrH/933-basic-e2e-smoke-tests

the tickets is now done but ill separate that bit from it which is not finished and it will be in order to do what you said